setup.py

```diff
@@ -306,13 +320,13 @@ def build_oneapi_backend():
     eca, ela, includes = get_build_options()
 
     return build_backend.build_cpp(
-        cc='dpcpp',
-        cxx='dpcpp',
+        cc='icx',
+        cxx='icx',
         sources=['src/oneapi/oneapi_backend.cpp'],
         targetname='oneapi_backend',
         targetprefix='' if IS_WIN else 'lib',
         targetsuffix='.dll' if IS_WIN else '.so',
-        libs=get_libs('daal') + ['OpenCL', 'onedal_sycl'],
+        libs=get_libs('onedal_dpc') + get_additional_libs(),
```

Comment:
Why two functions with same purpose are needed?

Reply:
I think these changes will not be required with the fpk updates

Reply:
Sure! It should be done on separate PR(s).

> I think these changes will not be required with the fpk updates